src/sentry/static/sentry/app/views/performance/trends/utils.tsx

```diff
@@ -381,14 +379,10 @@ export function movingAverage(data, index, size) {
 /**
  * This function applies a query to limit the results based on the trend type to being greater or less than 100% (depending on the type)
  */
-function getLimitTransactionItems(
-  trendFunction: TrendFunction,
-  trendChangeType: TrendChangeType
-) {
-  const aliasedPercentage = getTrendAliasedQueryPercentage(trendFunction.alias);
-  let limitQuery = aliasedPercentage + ':<1';
+function getLimitTransactionItems(trendChangeType: TrendChangeType) {
+  let limitQuery = 'trend_percentage():<1 t_score():>1';
   if (trendChangeType === TrendChangeType.REGRESSION) {
-    limitQuery = aliasedPercentage + ':>1';
+    limitQuery = 'trend_percentage():>1 t_score():<-1';
```

Comment:
would personally vote for +/- 2 as the default cutoff, +/- 1 corresponds to a super low 70% confidence level (http://uregina.ca/~gingrich/tt.pdf)

Reply:
:+1: There's an upcoming PR to replace this with the Low/Medium/High mentioned